tests/integration/test_cloudwatch.py

```diff
@@ -198,6 +198,21 @@ def test_multiple_dimensions(self, cloudwatch_client):
         assert metrics
         assert len(metrics) == len(namespaces) * num_dimensions
 
+    def test_describe_alarms_converts_date_format_correctly(self, cloudwatch_client):
+        alarm_name = "a-%s" % short_uid()
+        cloudwatch_client.put_metric_alarm(
+            AlarmName=alarm_name,
+            EvaluationPeriods=1,
+            ComparisonOperator="GreaterThanThreshold",
+        )
+        try:
+            result = cloudwatch_client.describe_alarms()
+            alarm = result["MetricAlarms"][0]
```

Comment:
This part might clash with other tests since it depends on a fully clean alarm state which we generally try to avoid. Could also be an issue when testing against an AWS account if it has other alarms defined.

`describe_alarm()` should have fields for AlarmName or AlarmNamePrefix (or similar) to only filter relevant matching alarms.

Reply:
true, good catch!